.github/workflows/release.yml

```diff
@@ -1,38 +1,31 @@
-on: 
+on:
   release:
-    types: [published]
-  # Use with only Tags
+    types: [published, edited]
   workflow_dispatch:
-
 name: Upload Artifact Bundle to Release
-env: 
-  DEVELOPER_DIR: "/Applications/Xcode_16.2.app/Contents/Developer"
-  SCIPIO_DEVELOPMENT: 1
-
+env:
+  DEVELOPER_DIR: '/Applications/Xcode_16.2.app/Contents/Developer'
 jobs:
   release:
-    name: Build and Upload Artifact Bundle
     runs-on: macos-15
     steps:
       - uses: actions/checkout@v4
-      - name: Resolve Dependencies
-        run: |
-          swift package resolve
-      - name: Build Binary
-        run: |
-          swift build --disable-sandbox -c release --arch arm64
-          swift build --disable-sandbox -c release --arch x86_64
-      - name: Get Current Tag
-        run: echo "TAG_NAME=${{ github.event.release.tag_name || github.ref_name }}" >> $GITHUB_ENV
-      - name: Make Artifact Bundle
-        run: |
-          swift package --allow-writing-to-package-directory generate-artifact-bundle \
-          --package-version ${{ env.TAG_NAME }} \
-          --executable-name scipio \
-          --build-config release \
-          --build-folder .build
+      - name: Build for arm64
+        run: swift build --disable-sandbox -c release --arch arm64 
+      - name: Build for x86_64
+        run: swift build --disable-sandbox -c release --arch x86_64
+      - uses: giginet/github-action-artifactbundle@v2
```

Comment:
Can't we create universal binary? Artifact bundle can set universal binary

Reply:
Good question, in some reason, Scipio can't build it with Xcode build mode. So we can't make an universal binary with SwiftPM.

I'll note the context on this line.

Reply:
ok, thanks!